sdk/core/Azure.Core/src/Shared/DelayStrategyInternal.cs

```diff
@@ -10,7 +10,7 @@ namespace Azure.Core
     /// <summary>
     /// Strategy to control delay behavior.
     /// </summary>
-    internal abstract class DelayStrategy
+    internal abstract class DelayStrategyInternal
```

Comment:
Curious: why do we have this class and the public class?

Reply:
These are internal types that will be switched over after the new type is shipped.

Reply:
I'm afraid I don't follow - can you help me understand why the public types can't be used now?

Reply:
Because this would require adding a project reference to Core in every mgmt library which we cannot do.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/35437

Reply:
Sorry, I'm still a little confused -if we change the name of this file, would we need to update the management plane libraries that depend on it, like is happening in the maps, etc, project files below?  

I'm also wondering, if we're planning to replace this internal type, would it make sense to have this type inherit from the final type to validate that functionality, rather than having them be independent?

Reply:
Well this file will be deleted, so we would need to remove the reference from any csprojs that reference it.

We can't have this type inherit from the final type because the signatures are different. But the functionality has already been validated in https://github.com/Azure/azure-sdk-for-net/pull/33040

Reply:
Discussed offline.